src/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\OverseasBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        $treeBuilder = new TreeBuilder('answear_overseas');
+
+        $treeBuilder->getRootNode()
+            ->children()
+                ->scalarNode('environment')->isRequired()->defaultValue(null)->end()
```

Comment:
This can be an [enum node](https://symfony.com/doc/current/components/config/definition.html#enum-nodes)